svir/dialogs/load_output_as_layer_dialog.py

```diff
@@ -443,6 +443,19 @@ def load_from_npz(self):
         # if self.npz_file is not None:
         #     self.npz_file.close()
 
+    def get_investigation_time(self):
+        try:
+            investigation_time = self.npz_file['investigation_time']
+        except KeyError:
+            investigation_time = None
+        else:
+            # check if it's integer
+            if numpy.equal(numpy.mod(investigation_time, 1), 0):
+                investigation_time = int(investigation_time)
+            else:
+                investigation_time = float(investigation_time)
+        return investigation_time
+
```

Comment:
It is better not to hide the fact that the investigation time is a float. Let it to appear as a float, since it is a float.